cmd/servicelog/post.go

```diff
@@ -87,14 +89,37 @@ func (o *PostCmdOptions) Validate() error {
 	return nil
 }
 
+func (o *PostCmdOptions) CheckServiceLogsLastHour() bool {
```

Comment:
nitpick: this could be private

```suggestion
func (o *PostCmdOptions) checkServiceLogsLastHour() bool {
```